PocketMC.RemoteControl/Web/styles.css

```
@@ -1591,17 +1663,636 @@ svg {
+  height: 24px;
+  color: var(--primary);
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.file-icon svg, .backup-icon svg {
+  width: 20px;
+  height: 20px;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)